src/index.html

```diff
@@ -7,8 +7,56 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
```

Comment:
```suggestion
    <link rel="stylesheet" href="style.css">
```

try to update path and redeploy your changes

Reply:
ok